### ISSUE TITLE
Console error in v10 Release 10.284

### ISSUE THREAD

**Issue by strongpauly** [Aedif/TokenVariants#78]
```
token-variants.mjs:703 Uncaught (in promise) LibWrapperPackageError: Error detected in module Token Variant Art.

This is *not* a libWrapper bug.

Find information about this module here: https://github.com/Aedif/TokenVariants
Report this issue here: https://github.com/Aedif/TokenVariants/issues

== Technical Details:
Detected by libWrapper.
Package ID= token-variants
Error= Can't wrap 'Token.prototype._refreshIcon', target does not exist or could not be found.

[Detected 1 package: token-variants]
    at new 🎁constructor (libWrapper-wrapper.js:111:11)
    at libWrapper-api.js:82:18
    at se (libWrapper-api.js:155:9)
    at Function.🎁register [as register] (libWrapper-api.js:525:14)
    at Object.initialize [as fn] (token-variants.mjs:530:16)
```

**Comment by Aedif** [comment]
Hi, this particular error has been fixed in release [3.6.1](https://github.com/Aedif/TokenVariants/releases/tag/3.6.1), however, note that the module has not yet been ported to v10. so you are likely to come across other problems if you continue using it.

v10 support is being slowly introduced

**Comment by strongpauly** [comment]
I can confirmed this has been fixed in 3.6.1.  I'm currently only using the module in my v10 test environment, will raise any further bugs I find, but so far it looks good.  Thanks!
